server/src/test/java/org/elasticsearch/index/query/ExistsQueryBuilderTests.java

```diff
@@ -115,7 +115,7 @@ public void testIllegalArguments() {
 
     public void testFromJson() throws IOException {
         String json =
-                "{\n" +
+            "{\n" +
```

Comment:
can you undo all indentation changes to this pull request?

Reply:
Done.